app/src/main/java/com/infomaniak/mail/data/models/calendar/CalendarEventResponse.kt

```diff
@@ -0,0 +1,74 @@
+/*
+ * Infomaniak Mail - Android
+ * Copyright (C) 2024 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.mail.data.models.calendar
+
+import com.infomaniak.lib.core.utils.Utils
+import io.realm.kotlin.types.EmbeddedRealmObject
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+class CalendarEventResponse : EmbeddedRealmObject {
+
+    //region Remote data
+    @SerialName("user_stored_event")
+    private var userStoredEvent: CalendarEvent? = null
+    @SerialName("user_stored_event_deleted")
+    var userStoredEventDeleted: Boolean = false
+    @SerialName("attachment_event")
+    private var attachmentEvent: CalendarEvent? = null
+    @SerialName("attachment_event_method")
+    private var _attachmentEventMethod: String? = null
+    //endregion
+
+    val attachmentEventMethod: AttachmentEventMethod?
+        get() = Utils.enumValueOfOrNull<AttachmentEventMethod>(_attachmentEventMethod)
```

Comment:
This is used in the PR where reply to calendar event is supported. Placing it here already was mainly to avoid having to bump the model multiple times if things go wrong

Reply:
We just have to manage that the PR will not be merged too soon, it's not a problem

Reply:
I moved the code to the branch that adds the reply button logic